libflux/src/core/semantic/convert.rs

```diff
@@ -1,10 +1,14 @@
 use crate::ast;
 use crate::semantic::fresh::Fresher;
 use crate::semantic::nodes::*;
+use crate::semantic::types;
 use crate::semantic::types::MonoType;
+use std::collections::BTreeMap;
+use std::collections::HashMap;
 use std::result;
 
 pub type SemanticError = String;
+pub type SemanticMap<K, V> = BTreeMap<K, V>;
```

Comment:
You should just be able to import this from the other file.

Reply:
✅